middleware/browse/browse_test.go

```diff
@@ -1,14 +1,16 @@
 package browse
 
 import (
+        "encoding/json"
+	"github.com/mholt/caddy/middleware"
```

Comment:
In the future, consider using gofmt and goimports to help organize and clean up the code generally. I can merge this in for now, but it would be good to have that in the future. :smile: